### DISCUSSION TITLE
Create Group

### DISCUSSION THREAD

**Discussion by staysellingcarts** [justLV/onju-voice#17] [Ideas]
I'm waiting for my circuit boards to come in, I just ordered them last week. Would anyone here like to collaborate by creating a discord server to work on adding features and improving this project?

**Comment by b-shaw** [comment]
Did you create a discord group?  I just got mine working and have comments & changes to report.

**Reply by staysellingcarts** [reply]
No one ever responded. I setup one using home assistant but couldn't get it to recognize voice. We can set one up still. What's your discord tag?

**Comment by b-shaw** [comment]
I'm wondering if  here, there, or one of the HA lists would be the best place for this discussion.  My firmware will be focused on Home Assistant,  basically a dumb speaker and mic for voice I/O and music.

**Reply by staysellingcarts** [reply]
I've always used discord as a communication platform. Using their apis can be great for when you're building out a community. Yes I have 5 boards. I've already uploaded the package to one and put it together. Had issues with it taking voice commands. I put the project on the back burner as no seemed to want to collaborate. I have high hopes for expanding on this project just trying to find more people who are also.

**Comment by b-shaw** [comment]
Did you get your boards?  How's your progress?  I don't see much discussion lately.

**Comment by b-shaw** [comment]
nm - see you set up HA - can you add some comments on that?  I'm debating on how to proceed now that I have basic functionality.
